webpack.config.js

```diff
@@ -25,6 +25,10 @@ const config = {
   output: {
     filename: "[name].js",
     path: pathLib.resolve(__dirname, "build"),
+    // Set the hashFunction to "xxhash64" to combat a 
+    // compatibility/vulnerability issue that was causing 
+    // the build to error.  
```

Comment:
What line length are you using for wrapping here? Could we stick with ~80 characters, which is what most comments use?